nautobot/docs/user-guide/feature-guides/contacts-and-teams.md

```diff
@@ -2,37 +2,32 @@
 
 ## How to Assign Contacts and Teams to Nautobot Objects
 
-In order to assign contacts/teams to a particular object, we first need to define `Status` and `Role` objects for contenttype `extras:contactassociation`. Once that is done, we can navigate to the detail view of the object. We will use a circuit in this example.
+In order to assign contacts/teams to a particular object, we first need to define `Status` and `Role` objects for the content type `extras:contactassociation`. Once that is done, we can navigate to the detail view of the object. We will use a circuit in this example.
 
 ![Circuit](./images/contact-and-team/circuit.png)
 
 At the top right corner, you should see a group of buttons and one of them should say **Add Contact**.
-
-![Circuit Add Contact Button](./images/contact-and-team/circuit_button.png)
-
+    ![Circuit Add Contact Button](./images/contact-and-team/circuit_button.png)
```

Comment:
This change (and the ones below) "feel" wrong to me - why are these being changed to be different from the "Circuit" image embedding above?

Reply:
I was getting some markdown lint errors like:
```
nautobot/docs/user-guide/feature-guides/contacts-and-teams.md:31:1: MD029: Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/2/3] (ol-prefix)
```
which I thought  had to do with the images within the list, in my troubleshooting, I added the indentation to remove the error. 

Now when I test again locally with linter, removing the spaces, it does not throw any error, so I am removing the extra spacing